internal/operators/ocs/consts.go

```diff
@@ -3,9 +3,25 @@ package ocs
 type ocsDeploymentMode string
 
 const (
+	// Aggregate CPU for compact mode is 36(including OCP), OCP requires 4 CPU per host on master, and 2 CPU for each disk
+	// so per host requires (36-(4*3)-(2*3))/3 = 6 CPU per host
+	CPUCompactMode int64 = 6
+
+	// Aggregate CPU for minimal mode is 24(including OCP), OCP requires 2 CPU per host on worker, and 2 CPU for each disk
+	// so per host requires (24-(2*3)-(2*3))/3 = 4 CPU per host
+	CPUMinimalMode int64 = 4
+
+	// Aggregate Memory for compact mode is 96(including OCP), OCP requires 8 GB per host, and 5 GB for each disk
```

Comment:
OCP masters require 16GB of RAM according to https://docs.openshift.com/container-platform/4.7/installing/installing_bare_metal/installing-bare-metal.html#minimum-resource-requirements_installing-bare-metal